### ISSUE TITLE
karma test fail on materialize.js v100.2

### ISSUE THREAD

**Issue by DetlefPeters** [Dogfalo/materialize#5217]
We use karma tests as part of our build process.
With the latest materialize.js v100.2, I get an error during karma tests:
```
PhantomJS 2.1.1 (Windows 8 0.0.0) ERROR
  TypeError: null is not an object (evaluating '_this2.$overlay[0].parentNode.removeChild')
  at bower_components/materialize/dist/js/materialize.js:2120
```
The failing code is within the `animateOut()` function of `modal.js`.
This error can be avoided by putting this line of code in an if-Block similar to the one in the `destroy()` function:
```
if (!!_this2.$overlay[0].parentNode) {
        _this2.$overlay[0].parentNode.removeChild(_this2.$overlay[0]);
      }
```
I suggest adding this if-Block to the `modal.js` source code so that materialize will pass karma tests.

Best regards,
  Detlef

**Comment by DetlefPeters** [comment]
No reaction to this issue for almost four weeks now...anything wrong with it?

**Comment by Dogfalo** [comment]
We will fix this, but it will be released in 1.0
